packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/__tests__/share-my-ads-modal.spec.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { toPng } from 'html-to-image';
+import html2canvas from 'html2canvas';
```

Comment:
replaced `html-to-image` library as there were issues in applying the styles once the user tries to download the qr code first time. Had tried various methods such as waiting for the styles to load, setting a timeout to wait for the user to wait to download (10 seconds) but still didnt work. Realised it was an issue with the function to convert it to an image.

However the quality of the downloaded image is slightly worse now, but QR code is still scannable